src/plugins/intel_cpu/src/memory_desc/cpu_memory_desc_utils.cpp

```diff
@@ -92,28 +97,17 @@ CpuBlockedMemoryDescPtr MemoryDescUtils::generateCpuBlockedMemoryDesc(const ov::
         blk_strides);
 }
 
-std::string MemoryDescUtils::dim2str(Dim dim) {
-    return dim == Shape::UNDEFINED_DIM ? "?" : std::to_string(dim);
+std::shared_ptr<MemoryDesc> MemoryDescUtils::makeDummyDesc(const MemoryDesc &desc, Dim dummyVal) {
+    auto dummyShape = makeDummyShape(desc.getShape(), dummyVal);
+    return desc.cloneWithNewDims(dummyShape.getStaticDims());
 }
 
-std::string MemoryDescUtils::dims2str(const VectorDims& dims) {
-    std::stringstream output;
-    output << "{";
-
-    if (!dims.empty()) {
-        auto itr = dims.begin();
-        do {
-            output << dim2str(*itr);
-        } while (++itr != dims.end() && output << ", ");
-    }
-
-    output << "}";
-    return output.str();
+std::shared_ptr<MemoryDesc> MemoryDescUtils::makeEmptyDesc() {
+    return std::make_shared<EmptyMemoryDesc>();
```

Comment:
```suggestion
    static empty_desc = std::make_shared<EmptyMemoryDesc>();
    return empty_desc;
```
If add `status = Defined` to the EmptyMemoryDesc constructor, such usage becomes thread safe.

Reply:
Also thought about that.
But even emptymemorydesc is supposed to be used as a constant read-only object, I propose to avoid multithreading access anyway, just to keep things simple.

Reply:
For the memory object it's more difficult as the `Memory` class isn't thread safe in opposite to the EmptyMemoryDesc. If we introduce a special `EmptyMemoryClass`, we can also switch to a singleton. So don't see any problem of using EmptyMemoryDesc as a singleton. And it doesn't introduce any problems for the user, while saves extra malloc/free calls.

Reply:
I think it is still double-edged sword.
Let's do it anyway.

Reply:
Implemented as a static